### PR TITLE
Add top 20 stale PRs

### PR DESCRIPTION
**Pull request by MiguelWeezardo**

### CODE REVIEW

File: .github/workflows/reports.yml [hunk 1]
```diff
@@ -42,7 +42,8 @@ jobs:
           docker logs trino
       - name: Execute queries
         run: |
-          ./bin/reports.sh reports/index.md "Trino CI/CD Reports" sql/{health,runs-queue-time-per-day,runs-duration-per-day,jobs-duration,flaky-tests}.sql
+          ./bin/reports.sh reports/ci-cd/index.md "Trino CI/CD Reports" sql/ci-cd/{health,runs-queue-time-per-day,runs-duration-per-day,jobs-duration,flaky-tests}.sql
```
Comment (nineinchnick) [inline .github/workflows/reports.yml:45]: If you're moving existing reports to a sub-dir, can you fix links in the README?

Comment (nineinchnick) [reply .github/workflows/reports.yml:45]: And maybe add a `reports/index.md` that would have a table of contents.

File: .github/workflows/sync.yml [hunk 2]
```diff
@@ -20,7 +20,7 @@ jobs:
       TRINO_DEST_SCHEMA: hive.v2
       TRINO_SRC_SCHEMA: github.default
       EMPTY_INSERT_LIMIT: 5
-      SYNC_TABLES: runs,jobs,steps,check_suites,check_runs,check_run_annotations,pulls,artifacts
+      SYNC_TABLES: runs,jobs,steps,check_suites,check_runs,check_run_annotations,pulls,pull_commits,artifacts
```
Comment (nineinchnick) [inline .github/workflows/sync.yml:23]: We might also need `reviews` and `review_comments`

**Review by bitweld — APPROVED** [review]
LGTM

File: sql/pr/stale-prs.sql [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+, current_timestamp - max(pull_commits.committer_date) AS time_since_push
+FROM
+  pulls
+LEFT JOIN
+  reviews ON pulls.number = reviews.pull_number
+LEFT JOIN
+  pull_commits ON pulls.number = pull_commits.pull_number
+WHERE pulls.owner = 'trinodb' AND pulls.repo = 'trino' AND pulls.state = 'open' AND NOT pulls.draft
+GROUP BY 1, 2, 3, 4
+ORDER BY untouched_for DESC
```
Comment (nineinchnick) [inline sql/pr/stale-prs.sql:17]: We might want to duplicate this into multiple reports, sorted by different columns, to see oldest PRs but also longest "running" ones, with largest `max(reviews.submitted_at) - pulls.updated_at`. And maybe the ones waiting longest after last review (order by time_since_review desc).

**Review by nineinchnick — APPROVED** [review]

**Pull request merged**